docs/protocol-methods.rst

```diff
@@ -552,6 +549,117 @@ Protocol version 1.0 returning an error as the result:
 
   "258: txn-mempool-conflict"
 
+blockchain.transaction.broadcast_package
+========================================
+
+Broadcast a package of transactions to the network (submitpackage). The package must consist of a child with its parents,
+and none of the parents may depend on one another. The package must be topologically sorted,
+with the child being the last element in the array.
+
+**Signature**
+
+  .. function:: blockchain.transaction.broadcast_package(raw_txs, verbose=false)
+
+  *raw_txs*
+
+    An array of raw transactions, each as a hexadecimal string.
+
+  *verbose*
+
+    Whether a verbose coin-specific response is required.
+
+**Result**
+
+  If *verbose* is :const:`false`:
+
+    A dictionary with the following keys:
+
+    * `success`
```

Comment:
I do wonder if it would make sense to also add support for `testmempoolaccept` (https://bitcoincore.org/en/doc/30.0.0/rpc/rawtransactions/testmempoolaccept/) to allow users to check whether their package would be accepted before they actually submit?

Reply:
> I would rather skip it for now.

Alright, fine by me.